src/routes/klageskjema/begrunnelse/autosaveProgress.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Element } from 'nav-frontend-typografi';
+import styled from 'styled-components/macro';
+import { Ellipsis } from '../../../styled-components/ellipsis';
+import Success from '../../../assets/images/icons/Success';
+import Popover, { PopoverOrientering } from 'nav-frontend-popover';
+
+export enum AutosaveStatus {
+    SAVING = 0,
+    SAVED = 1,
+    NOT_SAVED = 2
+}
+
+interface Props {
+    autosaveStatus: number | undefined;
+}
+
+const AutosaveContainer = styled.div`
+    margin-top: 16px;
+    color: #78706a;
+    text-align: right;
+    display: flex;
+    justify-content: flex-end;
+    div {
+        cursor: pointer;
+        > svg {
+            margin-right: 5px;
+        }
+        display: flex;
+        flex-flow: row wrap;
+        align-items: center;
+    }
+`;
+
+const AutosaveProgressIndicator = (props: Props) => {
+    const [anker, setAnker] = useState<(EventTarget & HTMLDivElement) | undefined>(undefined);
+
+    const togglePopover = (ankerEl: (EventTarget & HTMLDivElement) | undefined): void => {
+        setAnker(anker ? undefined : ankerEl);
+    };
+
+    return (
+        <>
+            <Popover
+                ankerEl={anker}
+                onRequestClose={() => setAnker(undefined)}
+                orientering={PopoverOrientering.OverHoyre}
+            >
+                <p style={{ padding: '1rem' }}>Vi lagrer endringene dine automatisk.</p>
+            </Popover>
+
+            <AutosaveContainer>
+                <div onClick={e => togglePopover(e.currentTarget)}>
+                    {props.autosaveStatus === AutosaveStatus.SAVING && (
```

Comment:
Kunne vi skrevet om dette patternet, `props.autosaveStatus === AutosaveStatus.SAVING && (`, til en funksjon i samme fil som bruker `if` og `return` i stedet (`getContent(status: AutosaveStatus) { if...return }`)?

Jeg synes bare det er mer lesbart, og vi slipper å kjøre alle sjekkene for alle tilfellene :)
Hva tenker du?